interface/src/index.html

```diff
@@ -70,6 +70,10 @@
                 Download
               </a>
             </li>
+
+            <login-button></login-button>
+            <profile-button></profile-button>
```

Comment:
Should the ng-if logic on profile-button go here? It initially seems odd to have the condition on whether or not the button should be displayed inside the button itself.